resharper/resharper-unity/src/CSharp/Feature/Services/QuickDoc/UnityEventFunctionQuickDocPresenter.cs

```diff
@@ -67,8 +68,12 @@ private XmlNode GetDetails(IDeclaredElement element)
                 description = myEventFunction.GetParameter(myParameterName)?.Description;
 
             var details = CreateMemberElement(element);
-            if (!string.IsNullOrWhiteSpace(description))
-                details.CreateLeafElementWithValue("summary", description);
+            if (string.IsNullOrWhiteSpace(description)) return details;
+            details.CreateLeafElementWithValue("summary", description);
+            if (!element.GetPsiServices().Solution.HasComponent<IXmlDocLinkAppender>()) return details;
+            var uri = element.GetPsiServices().Solution.GetComponent<UnityOnlineHelpProvider>().GetUrl(element);
+            element.GetPsiServices().Solution.GetComponent<IXmlDocLinkAppender>().AppendExternalDocumentationLink(uri, element.ShortName, details);
+
```

Comment:
Does `AppendExternalDocumentationLink` need to be an instance method on `IXmlDocLinkAppender`? Feels like it might be simpler as a static method on something like `XmlDocHtmlUtil`

Reply:
I use check for `IXmlDocLinkAppender` presence to add link for Rider and avoid adding it for Re#.